CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 ### Changed
 
 - Removed availability zones for `GermanyWestCentral` in `Azure`.
+- Removed the `--domain` flag on `Azure`.
```

Comment:
These should be under the `### Removed` section

Reply:
Right, thanks